ocamldoc/Makefile

```diff
@@ -384,25 +396,105 @@ test_texi:
 	$(MKDIR) $@
 	$(OCAMLDOC_RUN) -texi -sort -d $@ $(INCLUDES) odoc*.ml odoc*.mli
 
-stdlib_man/Pervasives.3o: $(OCAMLDOC) $(STDLIB_MLIS)
+stdlib_man/Stdlib.3o: $(OCAMLDOC) $(STDLIB_MLIS) $(STDLIB_CMIS)
 	$(MKDIR) stdlib_man
-	$(OCAMLDOC_RUN) -man -d stdlib_man $(INCLUDES) \
-	-t "OCaml library" -man-mini $(STDLIB_MLIS)
+	$(OCAMLDOC_RUN) -man -d stdlib_man -nostdlib -I stdlib_non_prefixed \
+	-t "OCaml library" -man-mini $(STDLIB_MLIS) \
```

Comment:
I'm guessing that sane platforms normally return these in alphabetical order already, but mine doesn't! `$(sort $(STDLIB_MLIS))` is better